configs/promtail/promtail-config.yml

```diff
@@ -0,0 +1,38 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
```

Comment:
Is it useful, is this file available to container?

Reply:
_"The positions block configures where Promtail will save a file indicating how far it has read into a file. It is needed for when Promtail is restarted to allow it to continue from where it left off."_

file not available in container and not used. Removing